internal/service/project/model_project_test.go

```diff
@@ -26,20 +25,30 @@ const (
 )
 
 var (
-	roles        = []string{"GROUP_DATA_ACCESS_READ_ONLY", "GROUP_CLUSTER_MANAGER"}
-	roleList, _  = types.ListValueFrom(context.Background(), types.StringType, roles)
-	teamRolesSDK = []admin.TeamRole{
+	roles              = []string{"GROUP_DATA_ACCESS_READ_ONLY", "GROUP_CLUSTER_MANAGER"}
+	roleList, _        = types.ListValueFrom(context.Background(), types.StringType, roles)
+	roleSet, _         = types.SetValueFrom(context.Background(), types.StringType, roles)
+	ipAddresses        = []string{"13.13.13.13"}
+	ipAddressesList, _ = types.ListValueFrom(context.Background(), types.StringType, ipAddresses)
+	empptyTFList, _    = types.ListValueFrom(context.Background(), types.StringType, []string{})
+	teamRolesSDK       = []admin.TeamRole{
 		{
 			TeamId:    conversion.StringPtr("teamId"),
 			RoleNames: &roles,
 		},
 	}
-	teamsDSTF = []*project.TfTeamDSModel{
+	teamsDSTF = []*project.TFTeamDSModel{
 		{
 			TeamID:    types.StringValue("teamId"),
 			RoleNames: roleList,
 		},
 	}
+	teamsTFSet, _ = types.SetValueFrom(context.Background(), project.TfTeamObjectType, []project.TFTeamModel{
```

Comment:
shouldn't we check errors?

Reply:
not concerned for creating the test value, however adjusted model functions to return diags and check in the unit tests that no errors are encountered.